### ISSUE TITLE
'Homeless' user in docker can't start clickhouse-client

### ISSUE THREAD

**Issue by filimonov** [yandex/ClickHouse#5245]
When starting docker with `--user id:gid` and if that id:gid doesn't exists in container (i.e. have no name nor home) clickhouse-client doesn't start with that exception: 
```
# Poco::Exception. Code: 1000, e.code() = 13, e.displayText() = Access to file denied: //.clickhouse-client-history (version 19.8.1.459)
```

**Comment by filimonov** [comment]
Workaround: 
```
export HOME=/tmp
```

**Comment by filimonov** [comment]
Similar:
```sql
:) show databases;
Code: 106. DB::ErrnoException: Cannot append history to file /home/ubuntu/.clickhouse-client-history, errno: 13, strerror: Permission denied
# clickhouse-client terminates... 
```

**Comment by filimonov** [comment]
#5431

**Comment by filimonov** [comment]
Fixed in 19.8
